### PR TITLE
Remove TutorialManager access from SolGame

### PR DESCRIPTION
**Pull request by NicholasBatesNZ**
Brings #279 back up to date with develop and fixes the lingering tutorial elements issue

### CODE REVIEW

**Comment by GooeyHub** [comment]
Uh oh, something went wrong with the build. Need to check on that

**Comment by Adrijaned** [comment]
@GooeyHub: ok to test

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/java/org/destinationsol/game/SolGame.java [hunk 1]
```diff
@@ -532,11 +531,15 @@ public void setRespawnState() {
...
+
     public SolApplication getSolApplication() {
         return solApplication;
     }
 
     public HullConfigManager getHullConfigManager() {
         return hullConfigManager;
     }
-}
+}
```
Comment (Adrijaned) [inline engine/src/main/java/org/destinationsol/game/SolGame.java]: Newline at the end of file

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**